tests/ops/test_qubit_ops.py

```diff
@@ -1172,12 +1208,6 @@ def test_multirz_generator(self, qubits, mocker):
         assert np.allclose(gen[0], expected_gen_mat)
         assert gen[1] == -0.5
 
-        spy = mocker.spy(qml.utils, "pauli_eigs")
-
-        op.generator
-        spy.assert_not_called()
-
-
```

Comment:
What is the motivation for removing this part? I think it's not strictly required in terms of test coverage, but was a nice addition to check that the generator is not reconstructed each time it is accessed.

Reply:
i see, previously when i saw this `op.generator`, I thought this was something that was forgotten to remove. no problem!